assets/src/edit-story/components/library/panes/media/media3p/providerTab.js

```diff
@@ -26,11 +26,20 @@ import { useKeyDownEffect } from '../../../../keyboard';
 import { useConfig } from '../../../../../app/config';
 
 const Tab = styled.span`
-  cursor: pointer;
+  display: inline-block;
+  border-radius: 100px;
+  background-color: ${({ active, theme }) =>
+    active ? theme.colors.fg.primary : 'transparent'};
   font-size: 16px;
   margin-right: 16px;
-  border-bottom: ${({ theme, active }) =>
-    active ? `solid 4px ${theme.colors.accent.primary};` : 'none'};
+  text-align: center;
+  opacity: 0.86;
+  cursor: pointer;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  padding: 6px 16px;
+  color: ${({ active, theme }) =>
+    active ? theme.colors.bg.workspace : theme.colors.bg.divider};
```

Comment:
These pills don't really seem to match the design:
In the editor:
<img width="367" alt="Screenshot 2020-08-20 at 18 15 17" src="https://user-images.githubusercontent.com/3294597/90790824-c6a27300-e308-11ea-9320-972b2f51bf57.png">

From Figma:
<img width="429" alt="Screenshot 2020-08-20 at 18 15 06" src="https://user-images.githubusercontent.com/3294597/90790830-c86c3680-e308-11ea-8908-82e501ccf2fa.png">

The category pills seem to be smaller (should be higher) in the implementation compared to Figma.

Also, would it make sense to use the same base component for these two? The text part at least seems to partially match (line-height, default color, etc.)

Reply:
Fixed the CSS in 422d83444 to better reflect the design. Actually the two pills are annoyingly different, but based them off a common css component now.